src/app/app.service.ts

```diff
@@ -118,4 +155,14 @@ export class AppService {
         }
         else return console.log('place name not valid');
     }
-}
+
+    conditionHighlight(wind:number, precip:number, vis:number) {
+    if (wind <= 9 
```

Comment:
Formatting here looks odd. Should be something like:

```
if (wind <= 9  && precip <= 20  && vis >= 9000) {
    return true;
} else {
    return false;
}
```

I'd also be tempted to move those numbers up into vars to make them easier to config / change if you ever needed to later.